cernopendata/modules/fixtures/data/records/data-policies.json

```diff
@@ -349,5 +349,51 @@
         "Policy"
       ]
     }
+  },
+  {
+    "abstract": {
+      "description": "This document defines the DELPHI data preservation, re-use, and open access policy as endorsed by the collaboration board in spring 2024."
+    },
+    "accelerator": "CERN LEP",
+    "authors": [
+      {
+        "name": "The DELPHI collaboration"
```

Comment:
BTW for conformance with the style of other collaborations, I'll remove the leading "The" when merging.